covid_moonshot_ml/datasets/utils.py

```diff
@@ -504,3 +504,116 @@ def get_ligand_RMSD_mdtraj(ref_fn, mobile_fn):
+    ['CC(C)NCCOc1ccc2cncc(c2c1)N3CCC4(C3=O)CN(Cc5c4cc(cc5)Cl)S(=O)(=O)CC6(CC6)C#N MAT-POS-1a788f51-3'],
+    ['c1ccc2c(c1)cncc2N3CC(C4(C3=O)CN(Cc5c4cc(cc5)Cl)S(=O)(=O)CC6(CC6)C#N)CC7CCCCC7 MIC-UNK-c85ea37c-1'],
+    ['CN(C)CC(COc1ccc2cncc(c2c1)N3CCC4(C3=O)CN(Cc5c4cc(cc5)Cl)S(=O)(=O)CC6(CC6)C#N)O EDJ-MED-ee81482e-3'],
+    ['c1cc2cncc(c2cc1CN3CC4(C3)COC4)N5CCC6(C5=O)CN(Cc7c6cc(cc7)Cl)S(=O)(=O)CC8(CC8)C#N EDJ-MED-ee81482e-5'],
+    ['CNC(=O)C1(CC1)N2CC3(CCN(C3=O)c4cncc5c4cc(cc5)OCCN6CCOCC6)c7cc(ccc7C2=O)Cl EDJ-MED-dfa1d800-2'],
+    ['CNC(=O)C1(CC1)N2CC3(CCN(C3=O)c4cncc5c4cc(cc5)OCCN6CC(C6)(F)F)c7cc(ccc7C2=O)Cl EDJ-MED-dfa1d800-5'],
+    ['CN1CCCC1COc2ccc3cncc(c3c2)N4CCC5(C4=O)CN(Cc6c5cc(cc6)Cl)S(=O)(=O)CC7(CC7)C#N MAT-POS-40ad851a-1'],
+    ['CCN(CC)CCOc1ccc2cncc(c2c1)N3CCC4(C3=O)CN(Cc5c4cc(cc5)Cl)S(=O)(=O)CC6(CC6)C#N MAT-POS-40ad851a-3'],
+    ['c1cc2cncc(c2cc1OCCN3CCCC3)N4CCC5(C4=O)CN(Cc6c5cc(cc6)Cl)S(=O)(=O)CC7(CC7)C#N EDJ-MED-468565e0-1'],
+    ['CN1CCC(CC1)Oc2ccc3cncc(c3c2)N4CCC5(C4=O)CN(Cc6c5cc(cc6)Cl)S(=O)(=O)CC7(CC7)C#N LUO-POS-d1147590-1'],
+    ['CNC(=O)C1(CC1)N2CC3(CCN(C3=O)c4cncc5c4cc(cc5)OCCNCC(F)(F)F)c6cc(ccc6C2=O)Cl EDJ-MED-4138fde9-6'],
+    ['CC(C)(C)NCCOc1ccc2cncc(c2c1)N3CCC4(C3=O)CN(Cc5c4cc(cc5)Cl)S(=O)(=O)CC6(CC6)C#N MAT-POS-1a788f51-4'],
+    ['c1cc2cncc(c2cc1OCCCN3CCCC3)N4CCC5(C4=O)CN(Cc6c5cc(cc6)Cl)S(=O)(=O)CC7(CC7)C#N MAT-POS-40ad851a-5'],
+    ['c1cc2cncc(c2cc1OCCN3CCOCC3)N4CCC5(C4=O)CN(Cc6c5cc(cc6)Cl)S(=O)(=O)CC7(CC7)C#N EDJ-MED-468565e0-2'],
+    ['c1cc2cncc(c2cc1OCCN3CC(C3)(F)F)N4CCC5(C4=O)CN(Cc6c5cc(cc6)Cl)S(=O)(=O)CC7(CC7)C#N EDJ-MED-468565e0-5'],
+    ['c1cc2cncc(c2cc1OCCNCC(F)(F)F)N3CCC4(C3=O)CN(Cc5c4cc(cc5)Cl)S(=O)(=O)CC6(CC6)C#N EDJ-MED-4138fde9-2'],
+    ['CN(C)c1ccc(cc1)C2CN(C(=O)C23CN(Cc4c3cc(cc4)Cl)S(=O)(=O)CC5(CC5)C#N)c6cncc7c6cccc7 MIC-UNK-c85ea37c-3'],
+    ['CNC(=O)C1(CC1)N2CC3(CCN(C3=O)c4cncc5c4cc(cc5)OCCN6CCS(=O)(=O)CC6)c7cc(ccc7C2=O)Cl EDJ-MED-dfa1d800-3'],
+    ['CNC(=O)C1(CC1)N2CC3(CCN(C3=O)c4cncc5c4cc(cc5)OCCN6CCC(CC6)(F)F)c7cc(ccc7C2=O)Cl EDJ-MED-dfa1d800-4'],
+    ['c1cc2cncc(c2cc1OCCN3CCS(=O)(=O)CC3)N4CCC5(C4=O)CN(Cc6c5cc(cc6)Cl)S(=O)(=O)CC7(CC7)C#N EDJ-MED-468565e0-3'],
+    ['c1cc2cncc(c2cc1OCCN3CCC(CC3)(F)F)N4CCC5(C4=O)CN(Cc6c5cc(cc6)Cl)S(=O)(=O)CC7(CC7)C#N EDJ-MED-468565e0-4'],
+    ['CCC(=O)Nc1ccc(cc1)C2CN(C(=O)C23CN(Cc4c3cc(cc4)Cl)S(=O)(=O)CC5(CC5)C#N)c6cncc7c6cccc7 MIC-UNK-c85ea37c-2']
+    ]
+    #### ^^ REPLACE WITH ExperimentalCompoundData PARSE INTO LIST OF SMILES^^
+
+    with open(smarts_queries, "r") as queries:
+        queries_reader = csv.reader(queries)
+        query_smarts = [ q[0] for q in queries_reader ]
+
+        if ignore_comment:
+            # only keep SMARTS queries that are not commented.
+            query_smarts = [ q for q in query_smarts if not q[0] == "#"]
+        else:
+            # some of the SMARTS queries are commented - remove these for now.
+            query_smarts = [ q if not q[0] == "#" else q[1:] for q in query_smarts]
```

Comment:
should these be flipped? if not, could you maybe rewrite the comment for the `ignore_comment` option to make it a bit more clear? as it's currently written, it sounds like `ignore_comment=True` means you ignore any leading `#` and add the SMARTS query, while `ignore_comment=False` means remove any commented SMARTS

Reply:
I think this kind of construct is inherently confusing; I adjusted the docstring to make it clearer (hopefully): https://github.com/choderalab/covid-moonshot-ml/pull/42/commits/82d7c0657a5b665338b4742f5fa896cc91fc10a0

Reply:
Maybe just rename to a longer boolean? i.e. if `drop_commented_smarts_strings == True`, then you ignore the queries that are commented, but if it is false, you don't `drop_commented_smarts_strings`. I thing the issue is that here "comment" could either refer to the syntax of using comments, or to the commented lines themselves.